### PR TITLE
Ordinals (inscription) course

### PR DESCRIPTION
**Pull request by yusufidimaina9989**

### CODE REVIEW

File: en/course-04/lesson-01/test [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (msinkec) [inline en/course-04/lesson-01/test:1]: Empty file?

File: en/course-04/lesson-01/chapter-01/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Chapter 1: Course overview
+
+In this course, you will learn what are Ordinals, and even learn how to inscribe images, text and bsv-20 both version 1 and 2.
+
+The course is very simple and you will developed a platform that will allow user to simply inscribe Text, images and bsv-20 tokens on both main and testnet.
+
+We will build in two steps:
+
+1. Ordinla basics
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-01/README.md:9]: typo

File: en/course-04/lesson-01/chapter-02/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Chapter 2: Understanding Ordinals and the Ordinals Protocol
+
+## Introduction to Ordinals:
+Ordinals are a category of numbers that denote the position or order of an element in a sequence. Examples include "first," "second," "third," and so forth. In the realm of blockchain technology, the term "ordinals" extends beyond its traditional usage to encompass a numbering scheme or protocol associated with organizing and identifying data within the blockchain.
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-02/README.md:4]: Maybe elaborate how Ordinals in bitcoin is just a system to assign ID's to satoshis. Then go on how these can be inscribed with some arbitrary chunk of data.

**Review by msinkec — APPROVED** [review]

**Comment by yusufidimaina9989** [comment]
@msinkec  thanks for the review, i have fix and update it

File: en/course-04/lesson-01/chapter-01/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Chapter 1: Course overview
+
+In this course, you will learn what are Ordinals, and even learn how to inscribe images, text and bsv-20 both version 1 and 2.
+
+The course is very simple and you will developed a platform that will allow user to simply inscribe Text, images and bsv-20 tokens on both main and testnet.
+
+We will build in two steps:
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-01/README.md:7]: two steps?

File: en/course-04/lesson-01/chapter-02/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+This process follows the familiar pushdata scheme, simplifying the inscription of ordinals compared to the two-step commit + reveal process on BTC.
+1Sat Protocol:
+
+## 1. Targeting Specific Satoshis:
+The Ordinals protocol extends into the 1Sat Protocol, focusing on targeting a specific satoshi within the blockchain.
+In practical terms, a range of Satoshis is often used to meet dust limits.
+
+## 2. No Dust Limit in Bitcoin SV:
+Bitcoin SV eliminates the concept of a dust limit, allowing for the streamlined inscription of a single satoshi.
+The protocol is aptly named "1Sat," emphasizing the capability to work with single satoshi outputs.
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-02/README.md:25]: "1Sat",

File: en/course-04/lesson-01/chapter-02/README.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## 3. Compatibility for Additional Sats:
+It's important to note that the 1Sat Protocol is designed with flexibility, easily accommodating support for additional satoshis in a compatible manner.
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-02/README.md:28]: Wdym by this?

File: en/course-04/lesson-01/chapter-02/README.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+# Chapter 2: Understanding Ordinals and the Ordinals Protocol
+
+## Introduction to Ordinals:
+Ordinals within the Bitcoin protocol serve as a systematic way to assign unique identifiers to satoshis, the smallest unit of Bitcoin. Similar to the traditional use of ordinals in denoting order or position, in the Bitcoin blockchain, they play a crucial role in organizing and identifying individual satoshis within the overall sequence of transactions.
+
+In essence, these ordinals act as a numbering scheme, allowing for the precise identification of each satoshi and its position within the blockchain. This system not only provides a structured approach to tracking transactions but also opens up opportunities for additional functionalities. For instance, these ordinals can be inscribed with arbitrary chunks of data, enabling the embedding of information alongside the transactional history.
+
+## BTC vs. Bitcoin SV: Ordinals Protocol:
+## 1. Divergence in Features:
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-02/README.md:9]: Make lower heading level (###)

Comment (msinkec) [reply en/course-04/lesson-01/chapter-02/README.md:9]: Do the same elsewhere.

File: en/course-04/lesson-01/chapter-02/README.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+## 2. Key Components:
+
+## BSV-20 V1:
+Required Fields: Protocol (p), Operation (op), Ticker (tick), Max Supply (max), Mint Limit (lim), Decimals (dec), and others.
+Optional Fields: Decimals (dec), Mint Limit (lim), and others as specified.
+
+## BSV-20 V2 (Tickerless Mode):
+New Fields for Deploy+Mint: Protocol (p), Operation (op), Symbol (sym), Icon (icon), Amount (amt), and Decimals (dec).
+3. Implied Transfers:
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-02/README.md:55]: Missing heading.

File: en/course-04/lesson-01/chapter-02/README.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+## BSV-20 V1:
+Implied transfers were supported, allowing the transfer of mint or transfer inscriptions without creating new transfer inscriptions.
+BSV-20 V2:
+
+Implied transfers are deprecated in V2, and this functionality will be discontinued at block height 807000.
+4. Content Type Requirement:
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-02/README.md:62]: Missing heading as well.

File: en/course-04/lesson-01/chapter-04/README.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
+# Chapter 4: Inscribe Image
+## Introduction:
+The OrdinalImage.tsx file contains a React component named OrdinalImage. 
+This component is designed to facilitate the inscription of images using the OrdiNFTP2PKH class from the scrypt-ord library. 
+Let's break down the key features of this component.
+
+So, i will take through with step by step explaination and you can also get the complete code at [Github](https://github.com/sCrypt-Inc/inscribe/blob/master/src/ordinalImage.tsx)
```
Comment (msinkec) [inline en/course-04/lesson-01/chapter-04/README.md:7]: We will go through a step by step explanation...

**Pull request merged**